metabase/metabase
# chore: rework qb template tags to css modules

[epic](https://github.com/metabase/metabase/issues/50983)

### Description

PR 5/?

Reworks QB template tags from emotion to css modules

### How to verify

create a native question, add params and check how it looks like

Snippet part is also affected 

https://github.com/user-attachments/assets/b6873bd2-3d25-4993-9350-718037524e12
